pkg/distributor/distributor_test.go

```diff
@@ -1012,12 +1012,15 @@ func TestDistributor_QueryStream_ShouldReturnErrorIfMaxChunkBytesPerQueryLimitIs
 	flagext.DefaultValues(limits)
 
 	// Prepare distributors.
+	// Use replication factor of 2 to always read all the chunks from both ingesters,
+	// this prevents us from getting chunks from a slow ingester and double counting chunks in the limiter.
 	ds, _, r, _ := prepare(t, prepConfig{
-		numIngesters:     3,
-		happyIngesters:   3,
-		numDistributors:  1,
-		shardByAllLabels: true,
-		limits:           limits,
+		numIngesters:      2,
+		happyIngesters:    2,
+		numDistributors:   1,
+		shardByAllLabels:  true,
+		limits:            limits,
+		replicationFactor: 2,
```

Comment:
Is there any value in keeping an RF=3 test case also? I don't think there is, but worth thinking about at least.

Reply:
> Is there any value in keeping an RF=3 test case also?

For this test I don't think there's much value.